### ISSUE TITLE
Opt out of FLOC

### ISSUE THREAD

**Issue by JosephMcArthur** [OAButton/discussion#1804]
We don't want to enable tracking, which now not only means avoiding cookies / third party services whenever we can but also paying attention to FLOC.

Seemingly, we can opt out:

https://paramdeo.com/blog/opting-your-website-out-of-googles-floc-network

@markmacgillivray there is some workers code there. Can we implement that for the API, if that is something that makes sense.
@sssoz, we'll need to figure out how to do this for CloudFlare pages too. I'm not sure how to atm. @markmacgillivray, maybe you'd know?

**Comment by markmacgillivray** [comment]
Yeah I already set headers directly on the workers, so will add that one. Don't know about for pages though. You can control headers via page rules on enterprise plan but that's very expensive. There may be a pages-specific way to do it though. I'll look over the weekend if I get time.

**Comment by markmacgillivray** [comment]
Added for the API

**Comment by markmacgillivray** [comment]
There's already a cloudflare community issue discussing getting cloudflare to add a page rule to control it:

https://community.cloudflare.com/t/blocking-google-flocs/257043/3

I'd guess they may do, given their usual privacy approach. But no comment yet that I can find.

**Comment by JosephMcArthur** [comment]
Thanks Mark.

I'd say we can likely wait it out with CF. As you say, it seems likely they'll do something so I don't mind waiting for them. This still is still in beta and affecting a small number of people.
